cypress/e2e/Login.cy.js

```diff
@@ -0,0 +1,33 @@
+describe('Login Page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/');
```

Comment:
i'm still seeing this test as broken - are we expecting different behavior?

Reply:
ahh Cypress refactor. Ill hit that now--